docs/logging-level-guidelines.md

```diff
@@ -0,0 +1,23 @@
+# Logging Level Guidelines
+
+We follow SLF4J logging levels, which are `TRACE`, `DEBUG`, `INFO`, `WARN`, `ERROR`, `FATAL`.
+
+Here comes the guidelines of each logging level.
```

Comment:
As a "don't do" recommendation, I suggest we add:

"Regardless of the logging level though, we should make sure to never log any PII (Personally identifiable information)."